app/middlewares/errors.js

```diff
@@ -5,7 +5,8 @@ const DEFAULT_STATUS_CODE = 500;
 
 const statusCodes = {
   [errors.DATABASE_ERROR]: 503,
-  [errors.DEFAULT_ERROR]: 500
+  [errors.DEFAULT_ERROR]: 500,
+  [errors.API_ERROR]: 400
```

Comment:
ApiError not always its a client error could be an external error too, for this case could be always 503 or something in that range 5xx